3_visualize/src/build_peaks_svg.R

```diff
@@ -5,33 +5,5 @@ build_peaks_svg <- function(out_file, data_in, sites_sf, svg_width, svg_height)
   
   add_background_map(svg_root, svg_width = svg_width, outline_states = FALSE)
   
-  # Add spark line within group per site
-  
-  # TODO: THIS ONLY WORKS FOR A SINGLE DAY RIGHT NOW. I CHOSE OCT 31, 2019
-  # TODO: Blanket removing any path with an NA in it. There is a smarter way 
-  # to do this but will come back to that.
-  data_in <- data_in %>% 
-    filter(Date == as.Date("2019-10-31")) %>% 
-    filter(!grepl("NA", path))
-  
-  sites <- sites_sf %>% pull(site_no) %>% unique
-  for(s in sites) {
-    site_coords_svg <- sites_sf %>%
-      filter(site_no == s) %>%
-      convert_coords_to_svg(svg_width = svg_width, view_bbox = st_bbox(generate_usa_map_data()))
```

Comment:
So you are no longer using the SVG with the sites already included? You are adding via D3?

Reply:
Yes I left some notes about that here https://github.com/USGS-VIZLAB/gw-conditions/issues/18#issuecomment-932971880

I think it's worth returning to this to draw the initial map from the pipeline, and then grab those paths with D3 for the animation. I was hitting some blocks with how it was set up though due to missing sites when there wasn't data on the first day, and the way I was modifying them with D3 (was creating duplicate elements). This PR addresses those issues but is slower to load.